CONTRIBUTING.md

```diff
@@ -112,9 +112,9 @@ Summary of our git branching model:
   code](http://www.codinghorror.com/blog/2008/07/coding-without-comments.html);
 - Name identifiers (variables, classes, functions, module names) with readable
   names (`x` is always wrong);
-- When manipulating strings, use [Python's new-style
-  formatting](http://docs.python.org/library/string.html#format-string-syntax)
-  (`'{} = {}'.format(a, b)` instead of `'%s = %s' % (a, b)`);
+- When manipulating strings, use [Python's f-string
+  formatting](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals)
+  (f`"{a} = {b}"` instead of `'{} = {}'.format(a, b)` or `'%s = %s' % (a, b)`);
```

Comment:
I believe these are a part of the extraneous changes that Steven saw in this PR.
Furthermore, I would add that the format-string syntax is still supported, but the f-string formatting is generally preferred.

Reply:
makes sense! thanks